packages/website/blog/2024-05-27-announcing-typescript-eslint-v8-beta.mdx

```diff
@@ -311,9 +307,81 @@ createRule({
 
 See [feat(utils): allow specifying additional rule meta.docs in RuleCreator](https://github.com/typescript-eslint/typescript-eslint/pull/9025) for more details.
 
+### Flat configuration `RuleTester`
+
+:::note
+This breaking change was missed in the initial versions of this blog post.
+:::
+
+The `RuleTester` provided by [`@typescript-eslint/rule-tester`](/packages/rule-tester) is a fork of ESLint's `RuleTester`.
+In typescript-eslint v7 and earlier, `RuleTester`'s constructor allowed providing legacy "eslintrc" options -- mirroring ESLint v8 and earlier.
+In typescript-eslint v8, `RuleTester`'s constructor now instead allows providing new "flat" config options -- mirroring ESLint v9.
+
+Per ESLint flat configs, any parser configurations you provide will need to be inside a `languageOptions` property:
+
+```ts title="rule.test.ts"
+import { RuleTester } from '@typescript-eslint/rule-tester
+
+const ruleTester = new RuleTester({
+  // Add this line
+  languageOptions: {
+    parserOptions: {
+      tsconfigRootDir: import.meta.dirname,
+    },
+  // Add this line
+  },
+});
+```
+
+Any `parser` you provide will need to be the parser itself, rather than a string name of the package:
+
+```ts title="rule.test.ts"
+import { RuleTester } from '@typescript-eslint/rule-tester
+// Add this line
+import jsoncParser from "jsonc-eslint-parser";
+
+const ruleTester = new RuleTester({
+  // Add this line
+  languageOptions: {
+    // Add this line
+    parser: jsoncParser,
+    // Remove this line
+    parser: "jsonc-eslint-parser",
+  // Add this line
+  },
+});
+```
+
+This change brings typescript-eslint's `RuleTester` in line with ESLint's `RuleTester` and flat config.
+In doing so, it changes two `parserOptions` defaults:
+
+- `ecmaVersion`: from `5` to `'latest'`
+- `sourceType`: from `'script'` to `'module'`
+
+If you were specifying either or both of those in your tests, you likely can now omit them:
+
+```ts title="rule.test.ts"
+import { RuleTester } from '@typescript-eslint/rule-tester
+
+// Add this line
+const ruleTester = new RuleTester();
+// Removed lines start
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: 'module',
+  },
+});
+// Removed lines end
+```
+
+For more details, see:
+
+- [ESLint > Migrate to 9.0.0 > `FlatRuleTester` is now `RuleTester`](https://eslint.org/docs/latest/use/migrate-to-9.0.0#-flatruletester-is-now-ruletester)
+- [typescript-eslint > feat(rule-tester): switched to flat config](https://github.com/typescript-eslint/typescript-eslint/pull/9603)
+
 ### Support for multi-pass fixes in `RuleTester`
 
-The `RuleTester` provided by `@typescript-eslint/rule-tester` is a fork of ESLint's `RuleTester`.
 One limitation of the original `RuleTester` is that it is not possible to verify the individual applied fixes when a rule provides multiple rounds of fixes.
```

Comment:
the wording "the original `RuleTester`" doesn't quite make sense without the now-deleted line about the rule tester being a fork of ESLint's RuleTester. Consider tweaking the wording to account for the deletion